pyproject.toml

```diff
@@ -36,21 +36,8 @@ flake8-max-line-length = 110
 flake8-max-doc-length = 79
```

Comment:
Please remove lines 33 to 36. They are not needed anymore.

Reply:
Done in the commit: [01dd3fe](https://github.com/lsst-ts/ts_m2gui/pull/37/commits/01dd3fe2079042833bf2522d2bc37fe5b1b6761a). Thanks!